### DISCUSSION TITLE
Show the invalid links detected by "All links use a valid format"

### DISCUSSION THREAD

**Discussion by andergmartins** [publishpress/publishpress-checklists#443] [Feature Requests]
The "All links use a valid format" check would be more useful if we know what (and possibly where) is the wrong link.
Maybe we can display the list of invalid links below the checklist?

**Comment by rizaardiyanto1412** [comment]
Its implemented on 2.16.0
https://github.com/publishpress/PublishPress-Checklists/issues/783
